plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/sql/SQLUtils.java

```diff
@@ -221,6 +231,23 @@ public static String makeSQLLike(String like)
         return like.replace("*", "%").replace("?", "_");
     }
 
+    public static String makeGlobFromSqlLikePattern(String sqlLikePattern) {
+
+        StringBuilder result = new StringBuilder();
+        for (int i = 0; i < sqlLikePattern.length(); i++) {
+            char charAtI = sqlLikePattern.charAt(i);
+            if (i + 1 < sqlLikePattern.length() && charAtI == '\\'
+                && (sqlLikePattern.charAt(i + 1) == '_' || sqlLikePattern.charAt(i + 1) == '%')) {
+                result.append(sqlLikePatternToGlobSymbolsMap.get(
+                    String.format("%c%c", charAtI, sqlLikePattern.charAt(i + 1))));
+                i++;
+            }
```

Comment:
This condition only checks for `\\_` and `\\%` sequences. Is this expected? If so, I'd rather remove the map lookup.

Reply:
not only. In the map also is a 3 another values

Reply:
No, this particular piece of code will only result in either of these two because your key is two characters long while others are one character long.